CPAC/resources/configs/pipeline_config_blank.yml

```diff
@@ -203,13 +203,12 @@ surface_analysis:
   # select those 'Freesurfer-' labeled options further below in anatomical_preproc.
   freesurfer:
 
+    run_reconall: Off
+
     # Add extra arguments to recon-all command
     reconall_args:
 
-    # (Optional) Provide an already-existing FreeSurfer output directory to ingress already-computed surfaces
-    freesurfer_dir:
-
-    # If anatomical_preproc['brain_extraction']['using'] includes FreeSurfer-ABCD and this switch is On, C-PAC will automatically turn this switch Off to avoid running FreeSurfer twice unnecessarily
+    # Run recon-all post-processing steps
     run: Off
```

Comment:
What's the difference between `run_reconall` and `run` here? I think we need some clarifying comments if they are different things, and I think these changes necessitate an update to the participant pipeline schema.

Otherwise this looks awesome!

Reply:
Thanks - I just updated it. I still think we need to come up with a better name, as `postproc` is used later as well.

`run_reconall`: will run Freesurfer's recon-all, the 24h+ step

`run_postproc`: will run the nodes which convert .mgz files to .nii, and the ones that generate tissue masks, etc. - this will enable the block that reads in Freesurfer outputs, either directly from recon-all (if run), or from the ingressed Freesurfer outputs

Reply:
Note: if you leave `run_reconall: On` but provide a Freesurfer directory (via ingress), it will still skip recon-all. So this doesn't have to be toggled `Off` everytime.

Reply:
We'll need to update this (I think from `run` to `run_reconall`) (or maybe we can drop this automation if it'll skip now in the case where it was previously running twice?):
https://github.com/FCP-INDI/C-PAC/blob/46b00a1aee998e18b1480f88bcb2418aef35d1b7/CPAC/surface/globals.py#L2-L5 https://github.com/FCP-INDI/C-PAC/blob/46b00a1aee998e18b1480f88bcb2418aef35d1b7/CPAC/utils/configuration/yaml_template.py#L99-L101 https://github.com/FCP-INDI/C-PAC/blob/46b00a1aee998e18b1480f88bcb2418aef35d1b7/CPAC/utils/configuration/yaml_template.py#L119-L122 https://github.com/FCP-INDI/C-PAC/blob/46b00a1aee998e18b1480f88bcb2418aef35d1b7/CPAC/utils/configuration/yaml_template.py#L145-L150 https://github.com/FCP-INDI/C-PAC/blob/46b00a1aee998e18b1480f88bcb2418aef35d1b7/CPAC/resources/configs/pipeline_config_default.yml#L199-L202

Reply:
(I wouldn't have remembered this, but the CI caught it)